src/memory_data.c

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <inttypes.h>
+
+#include "azure_c_shared_utility/xlogging.h"
```

Comment:
Not needed.

Reply:
Removed

---
In reply to: [248874282](https://github.com/Azure/azure-c-shared-utility/pull/259#discussion_r248874282) [](ancestors = 248874282)